geometry_msgs/PoseStamped[] posesStamped
int32 parent
int32 id
---
float64 quality